### DISCUSSION TITLE
Plotting on different computer

### DISCUSSION THREAD

**Discussion by ludwigmeier** [Chia-Network/chia-blockchain#4194] [Support]
Hi,
I have a windows machine with full node. And different Linux machine running as plotting computer. After finishing I transfer the files via ftp from Linux to windows main machine which act as farm.
the Plots are all marked as „failed to open, invalid plots“.
Can someone help me? Any Idea for reason?
Best

**Comment by VictorLaszio** [comment]
Did you try restarting Chia?
Normally that does the trick and removes "failed to open, invalid plots“

**Reply by ludwigmeier** [reply]
Will try again. Have to transfer it again.
will report asap it is done.

mit this the correct way?
I have the main computer windows 
And Linux plotting machines.

i run generate keys on Linux.
Then I runned this command 
chia plots create -k 32 -n 5 -b 12000 -f XXX -p YYY -t F:\TEMP -d G:\Plots -r 4 -e

with -p and -f with keys from windows main machine, is that correct?

**Reply by ludwigmeier** [reply]
Did restarted chia but still same error failed to open...

is the plotting command correct? And is a ftp transfer ok for transport?

**Comment by DarkSall3** [comment]
I was thinking to do the same but i'm still looking info on how to  move Plots .
alternatively  i will try this
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
let me know in this discussion if you find a solution bro ^_^

**Reply by ludwigmeier** [reply]
My solution was to enable binary mode inside ftp transfer. Now it is working fine

**Reply by BlckhtX64** [reply]
You are doing an error, that sometimes is so difficult to identify. The main error is when you're plotting, you select always a final directoy. When the plot is 100% fullfilled, automatically it starts farming, then, you send via FTP the same plot to the windows machine, to farm it again. Automatically when you load your plot, Chia software will give you an error because this plots is  already  farming on the linux machine. What do you need to do before send this plot to the window machine, is, once your plot is 100% completed and stored ( In another folder different to the windows machine (-no ftp address)) you need to cancel the directory (three points on 'add plot' button and remove the directory). By this way, Chia on the linux machine, will not find any plot to farm. Then you sent finally your complete plot via FTP to the Windows machine, and the error will not occurs any more. The old error message will remain until you restart the Chia soft.

**Reply by DarkSall3** [reply]
Thank you pal i'll try this trick ^_^

**Reply by BlckhtX64** [reply]
You're welcome :)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
